src/appleseed-max-impl/appleseedinteractive/interactivesession.cpp

```diff
@@ -71,6 +76,10 @@ void InteractiveSession::render_thread()
 
     // Render the frame.
     renderer->render();
+
+#if MAX_RELEASE == MAX_RELEASE_R17
+    static_cast<AppleseedInteractiveRender*>(m_renderer)->AbortRender();
```

Comment:
It's a bit weird to call `AbortRender()` at the end of the render. Do you really need to do anything? Wouldn't calling `EndSession()` be more logical?